docs/content/docs/ops/state/state_backends.md

```diff
@@ -325,6 +325,126 @@ public class MyOptionsFactory implements ConfigurableRocksDBOptionsFactory {
 
 {{< top >}}
 
+## Enabling Changelog
+
+// todo: Chinese version of all changed docs
+
+// todo: mention in [large state tuning]({{< ref "docs/ops/state/large_state_tuning" >}})? or 1.16?
+
+{{< hint warning >}} The feature is in experimental status. {{< /hint >}}
+
+{{< hint warning >}} Enabling Changelog may have a negative performance impact on your application (see below). {{< /hint >}}
+
+### Introduction
+
+Changelog is a feature that aims to decrease checkpointing time, and therefore end-to-end latency in exactly-once mode.
+
+Most commonly, checkpoint duration is affected by:
+
+1. Barrier travel time and alignment, addressed by
+   [Unaligned checkpoints]({{< ref "docs/ops/state/checkpointing_under_backpressure#unaligned-checkpoints" >}})
+   and [Buffer debloating]({{< ref "docs/ops/state/checkpointing_under_backpressure#buffer-debloating" >}})
+2. Snapshot creation time (so-called synchronous phase), addressed by Asynchronous snapshots
+3. Snapshot upload time (asynchronous phase)
+
+The latter (upload time) can be decreased by [Incremental checkpoints]({{< ref "#incremental-checkpoints" >}}). However,
+even with Incremental checkpoints, large deployments tend to have at least one task in every checkpoint that uploads a
+lot of data (e.g. after compaction).
```

Comment:
1. Add some context here:
Current Incremental Checkpoints depend on the implementation of different types of state backends. For example, for rocksdb, compaction happens when ... 

2. Explain a bit why compaction is bad
Compaction may cause more data to be uploaded, and more time to upload....

Reply:
1. I think explaining when the compaction can be triggered is outside of scope of this section. There are several compaction algorithms, to start with.
2. Agree

Reply:
I am not asking to explain the entire different compaction algorithms, but since you mentioned "at least one task in every checkpoint that uploads a lot of data (e.g. after compaction)." You need to explain why.

That's what I mean by providing some context: what causes to upload a lot data; that's compaction, then why compaction causes more data to upload... e.t.c

I do not think people can infer directly why compaction can cause more data to upload until you explain at least a little bit of different level of compaction e.t.c.

Reply:
I've expanded this section to address (2), do you still have any concerns about (1)?